OpenSim/Common/Storage.cpp

```diff
@@ -123,44 +163,74 @@ Storage::Storage(int aCapacity,const string &aName) :
     setName(aName);
 }
 //_____________________________________________________________________________
-/**
+/*
  * Construct an Storage instance from file.
- * This constructor is far from bullet proof.
  *
- * @param aFileName Name of the file from which the Storage is to be
+ * @param fileName Name of the file from which the Storage is to be
  * constructed.
  */
```

Comment:
The doxygen comment should say that Storage can now read non-STO files (e.g., C3D), and the doxygen block should be moved to the header.

Reply:
I will add comments to the effect that this ability exists but point readers to using the `FileAdapter` and tables for reading and using the data.